README.md

```diff
@@ -582,6 +582,17 @@ Previous releases may also have been signed with one of the following GPG keys:
 Information on the current Node.js Working Groups can be found in the
 [TSC repository](https://github.com/nodejs/TSC/blob/master/WORKING_GROUPS.md).
 
+
+### Strategic Initiatives
+
+At any one time the Node.js project has a number of strategic initiatives
```

Comment:
I'd recommend removing this entire paragraph.The first sentence is equivalent to "Strategic Intiaitves are a thing that exist" but doesn't tell you anything about what they are. The second sentence is a bit better, mentioning champions, but IMO that's non-essential information for the README. Maybe we can just include a link to the Strategic Initiatives somewhere in the doc where links to other important docs are and call it good.

Reply:
@Trott it already exists in the TSC repo.  I opened this PR as the discussion in the original PR suggested it might make sense to have it in the core README.md for visibility.  I don't really have a strong feeling one way or the other ...

Reply:
@MylesBorins what do you think as you suggested I add it here (I think)